src/backend/llm/test_llm.py

```diff
@@ -0,0 +1,16 @@
+test_text = (
```

Comment:
The text is in a complete different style. We should test it with the actual SRT format or the json object. I would assume the SRT format is sufficient enough. I will test it and if it is still showing promising output, we can use this. Otherwise, we should change the system message until it is acceptable.